lib/models/team_model.dart

```
@@ -1,6 +1,7 @@
 class LightTeam {
-  LightTeam(this.id, this.number, this.name);
+  LightTeam(this.id, this.number, this.name, this.colorsIndex);
   final int id;
   final int number;
   final String name;
+  final int colorsIndex;
```

A more elegant approach would be to have a `final Color color` field here.
A constructor should translate the index to a real `Color` (and fail loudly if the index is invalid).